src/main/java/array/Exercise05.java

```diff
@@ -0,0 +1,23 @@
+package array;
+
+/**
+ * Exercise05: A whole number of n elements values from 1 to n, removing any value and then randomly arranged.
+ * Write a program to receive the integer array of the above process and search for the number that has been removed.
+ */
+public class Exercise05 {
+    public int findMissingNumber(int[] arr) {
+        int n = arr.length + 1;
```

Comment:
```suggestion
        final int n = arr.length + 1;
```
Always use final for variables that are not meant to change.